content/how-to/account-management/account-management-faq/index.md

```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
 type: article
 created_date: '2019-05-16'
 created_by: Stephanie Fillmon
-last_modified_date: '2020-09-08'
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2020-07-15'
```

Comment:
Remember to update the date right before merge

Reply:
Date change